### PR TITLE
[ITK v4.13.1] without VTK6

### PR DESCRIPTION
**Pull request by mathildemerle**
Same work as in https://github.com/Inria-Asclepios/medinria-superproject/pull/28 
Trello https://trello.com/c/FD7oMaYr/116-tester-itk-412

I wanted to try ITK 4.12 and end up switching to ITK 4.13.1 (the last one) without needing to switch to VTK6 \o/, with a small patch on TTK.

 * ITK 4.13.1, no patch
 * RPI on master branch
 * TTK still on music branch, with a patch

:m:

### CODE REVIEW

**Comment by juliencastelneau** [comment]
Please
1)  Add a patch in qtDCM : 
   - Code/CMakeLists => Add ITKIOBruker & ITKIOMINC in target_link_libraries 

2) Same modification on CmakeLists in medInria  : 
  - src/medImageIO 
  - src/medRegistration
  - src/medVtkInria

**Comment by mathildemerle** [comment]
Done here and https://github.com/Inria-Asclepios/medInria-public/pull/483

Strange i did not have this bugs. Could be an OS problem, maybe.

File: cmake/externals/projects_modules/ITK.cmake [hunk 1]
```diff
@@ -84,7 +84,7 @@ set(cmake_args
 ## Check if patch has to be applied
 ## #############################################################################
 
-ep_GeneratePatchCommand(ITK ITK_PATCH_COMMAND ITK_Mac_Rpath.patch)
+#ep_GeneratePatchCommand(ITK ITK_PATCH_COMMAND ITK_Mac_Rpath.patch)
```
Comment (juliencastelneau) [inline cmake/externals/projects_modules/ITK.cmake]: Maybe we could remove this line.
But if you prefer, we could keep it in comments.

**Review by juliencastelneau — APPROVED** [review]

File: cmake/externals/projects_modules/ITK.cmake (hunk 1 above)
Comment (mathildemerle) [reply cmake/externals/projects_modules/ITK.cmake]: Done :)

**Review by paulineMig — APPROVED** [review]

**Pull request merged**